server.js

```diff
@@ -51,14 +48,29 @@ app.get('/api/*', (req, res) => {
 })
 
 app.get('/*', (req, res) => {
+  const store = configureStore()
+
   match({ history, routes, location: req.url }, (err, redirect, props) => {
-    const appHtml = renderToString(
-      <Provider store={store}>
-        <RouterContext {...props} />
-      </Provider>
-    )
-    const page = appShell.replace('<div id=\'app\'></div>', `<div id='app'>${appHtml}</div>`)
-    res.send(page)
+    if (err) {
+
+      res.status(500).send('Internal Server Error')
```

Comment:
This makes sense, though I wonder if we want to pass the error along for easier debugging?